### PR TITLE
Content Service Fixes

### PR DESCRIPTION
**Pull request by blainsmith**
Pay close attention to efde266 for anon users. I will start another branch for dealing with the new `/config` JWT response, etc.

### CODE REVIEW

File: lib/services/identity/controllers/identity-config-controller.js [hunk 1]
```diff
@@ -17,6 +19,24 @@ class IdentityConfigController {
 		this.bus
 			.query({role: 'identity', cmd: 'config'}, {channel, platform, user})
 			.then(resource => {
+				if (user) {
+					return resource;
+				} else { // eslint-disable-line no-else-return
```
Comment (kixxauth) [inline lib/services/identity/controllers/identity-config-controller.js]: It's cleaner to follow eslint no-else-return here, no? #nitpick

File: lib/services/identity/controllers/identity-config-controller.js [hunk 2]
```diff
@@ -17,6 +19,24 @@ class IdentityConfigController {
...
+					const id = uuid.v4();
+					const user = {id, channel: channel.id, type: 'user'};
+					const payload = {channel: channel.id, platform: platform.id, user: id, audience: ['platform']};
+
+					return Promise.join(
+						this.bus.sendCommand({role: 'store', cmd: 'set', type: 'user'}, user),
+						this.bus.query({role: 'identity', cmd: 'sign'}, payload),
+						(user, jwt) => {
+							resource.jwt = jwt;
```
Comment (kixxauth) [inline lib/services/identity/controllers/identity-config-controller.js]: Maybe add `resource.user = user` here ? I added to the config a while back, so just to be consistent.

File: package.json [hunk 3]
```diff
@@ -30,7 +30,8 @@
     "lodash": "~3.10.0",
     "node-uuid": "~1.4.0",
```
Comment (brkattk) [inline package.json]: We are already using node-uuid in this project. IIRC it does basically the same thing as uuid. I believe we use node-uuid in other projects as well

File: lib/services/identity/controllers/identity-config-controller.js (hunk 1 above)
Comment (blainsmith) [inline lib/services/identity/controllers/identity-config-controller.js]: It complains with the no return OR !user negation. Either way its going to complain so I went with readable positive case.

**Comment by blainsmith** [comment]
I'll update the rest. Good review!

**Pull request merged**